docs/generators/aspnetcore.md

```diff
@@ -16,14 +16,18 @@ sidebar_label: aspnetcore
 |packageVersion|C# package version.| |1.0.0|
 |packageGuid|The GUID that will be associated with the C# project| |null|
 |sourceFolder|source folder for generated code| |src|
-|compatibilityVersion|ASP.Net Core CompatibilityVersion| |Version_2_1|
-|aspnetCoreVersion|ASP.NET Core version: 2.2 (default), 2.1, 2.0 (deprecated)| |2.2|
+|compatibilityVersion|ASP.Net Core CompatibilityVersion| |Version_2_2|
+|aspnetCoreVersion|ASP.NET Core version: 3.0 (preview4 only), 2.2 (default), 2.1, 2.0 (deprecated)| |2.2|
+|swashbuckleVersion|Swashbucke version: 3.0.0 (default), 4.0.0| |3.0.0|
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
 |useDateTimeOffset|Use DateTimeOffset to model date-time properties| |false|
 |useCollection|Deserialize array types to Collection&lt;T&gt; instead of List&lt;T&gt;.| |false|
 |returnICollection|Return ICollection&lt;T&gt; instead of the concrete type.| |false|
 |useSwashbuckle|Uses the Swashbuckle.AspNetCore NuGet package for documentation.| |true|
 |isLibrary|Is the build a library| |false|
+|isFramework|Use frameworkReference for ASP.NET Core 3.0+ and  PackageReference  ASP.NET Core 2.2 or earlier.| |false|
+|useNewtonsoft|Uses the Newtonsoft JSON library.| |true|
+|useDefaultRoutng|Use default routing for the  ASP.NET Core version. For 3.0 turn off default because it is not yet supported.| |true|
```

Comment:
If "default routing" isn't supported in 3.0, I would think that this shouldn't be user-configurable until it's supported.

Reply:
I have put it in prep for the future and also because the support for 2.1 needed the configuration.